### PR TITLE
[FIX] product_cost_usd: fix models, views and manifest t#53862

### PR DESCRIPTION
**Pull request by agarzaarvizu**
## This PR refers to the task[#53862](https://www.vauxoo.com/web#action=1928&cids=1&id=53862&menu_id=1490&model=project.task&view_type=form)

- Rename model files and remove parameters in super
- Rename views files and views IDs
- Change category and fix version in manifest
- Remove headers on files

### CODE REVIEW

**Comment by agarzaarvizu** [comment]
@luisg123v Done

File: product_cost_usd/__manifest__.py [hunk 1]
```diff
@@ -1,26 +1,20 @@
...
-# Copyright 2017 Vauxoo
-# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
-
 {
     "name": "Product Price Cost in USD",
     "summary": """
 This module adds the field Cost in USD to the Product form.
     """,
-    "version": "10.0.0.0.1",
+    "version": "11.0.0.0.1",
```
Comment (luisg123v) [inline product_cost_usd/__manifest__.py]:
```suggestion
    "version": "11.0.1.0.0",
```

File: product_cost_usd/models/product_pricelist.py [hunk 2]
```diff
@@ -32,7 +27,7 @@ def _compute_price_rule(self, products_qty_partner, date=False, uom_id=False):
         :param datetime date: validity date
         :param ID uom_id: intermediate unit of measure
         """
-        results = super(Pricelist, self)._compute_price_rule(products_qty_partner, date=date, uom_id=uom_id)
+        results = super()._compute_price_rule(products_qty_partner, date=date, uom_id=uom_id)
         usd_currency = self.env.ref("base.USD")
         for product_id in results:
             # get current price and pricelist item for product_id
```
Comment (luisg123v) [inline product_cost_usd/models/product_pricelist.py:33]: Move to separated file, as pricelist items may be viewed standalone

**Review by luisg123v — CHANGES_REQUESTED** [review]

**Comment by luisg123v** [comment]
@agarzaarvizu task ID seems to be wrong

**Comment by agarzaarvizu** [comment]
@luisg123v Done

**Review by luisg123v — APPROVED** [review]

**Pull request merged**